scripts/bash/run_enceladus.sh

```diff
@@ -381,37 +385,65 @@ if [ "$DRA_ENABLED" = true ] ; then
     add_spark_conf_cmd "spark.dynamicAllocation.enabled" "true"
     add_spark_conf_cmd "spark.shuffle.service.enabled" "true"
     add_spark_conf_cmd "spark.sql.adaptive.enabled" "true"
-    add_spark_conf_cmd "spark.dynamicAllocation.maxExecutors" ${DRA_MAX_EXECUTORS}
-    if [ ! -z "$DRA_MIN_EXECUTORS" ]; then
-        add_spark_conf_cmd "spark.dynamicAllocation.minExecutors" ${DRA_MIN_EXECUTORS}
+    add_spark_conf_cmd "spark.dynamicAllocation.maxExecutors" "${DRA_MAX_EXECUTORS}"
+    if [ -n "$DRA_MIN_EXECUTORS" ]; then
+        add_spark_conf_cmd "spark.dynamicAllocation.minExecutors" "${DRA_MIN_EXECUTORS}"
     fi
-    if [ ! -z "$DRA_ALLOCATION_RATIO" ]; then
-        add_spark_conf_cmd "spark.dynamicAllocation.executorAllocationRatio" ${DRA_ALLOCATION_RATIO}
+    if [ -n "$DRA_ALLOCATION_RATIO" ]; then
+        add_spark_conf_cmd "spark.dynamicAllocation.executorAllocationRatio" "${DRA_ALLOCATION_RATIO}"
     fi
-    if [ ! -z "$ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE" ]; then
-        add_spark_conf_cmd "spark.sql.adaptive.shuffle.targetPostShuffleInputSize" ${ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE}
+    if [ -n "$ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE" ]; then
+        add_spark_conf_cmd "spark.sql.adaptive.shuffle.targetPostShuffleInputSize" "${ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE}"
     fi
 fi
 
 JVM_CONF="spark.driver.extraJavaOptions=-Dstandardized.hdfs.path=$STD_HDFS_PATH \
 -Dspline.mongodb.url=$SPLINE_MONGODB_URL -Dspline.mongodb.name=$SPLINE_MONGODB_NAME -Dhdp.version=$HDP_VERSION \
 $MT_PATTERN"
 
+if [ "$HELP_CALL" == "1" ]; then
```

Comment:
**Question:* `--help` erases `--dry-run`. Is this what we want? In other words which one of these switches take precedence.

Reply:
Great question, I think it is ok. I either want to see the command line that is going to be run or I want to see what are my options.

Interestingly the help shou be even bigger since you can specify --jar, --class, and many more.

Reply:
> 
> Interestingly the help shoul be even bigger since you can specify --jar, --class, and many more.

Many more? And yeah I would add them...  🏰